manifest-editor/src/components/organisms/GridView/GridView.styles.tsx

```diff
@@ -6,14 +6,14 @@ export const GridViewContainer = styled.div<{ strip?: boolean }>`
   justify-content: space-between;
   align-items: center;
   height: 80vh;
+  margin-right: -40px;
```

Comment:
where did the 40px come from?

Reply:
@stephenwf Line 53, the whitespace created by the hidden menu item

Reply:
I think it should be `-2rem` to offset the menu button.